src/main/java/org/apache/commons/imaging/formats/jpeg/iptc/IptcRecord.java

```diff
@@ -50,6 +50,10 @@ public IptcRecord(final IptcType iptcType, final String value) {
         this.value = value;
     }
 
+    public IptcRecord(final IptcType iptcType, final String value) {
+        this(iptcType, value, "ISO-8859-1");
```

Comment:
+1 there is a constant defined in another class, but I wonder if we could use `StandardCharsets` as `commons-imaging` is using Java1.7+.